index.html

```diff
@@ -20,18 +20,14 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
 
     <!-- Imports an icon to represent the document. -->
-    <link rel="icon" href="assets/icons/icon_24.png" type="image/x-icon" />
+    <link rel="icon" href="icons/icon_24.png" type="image/x-icon" />
 
     <!-- Imports the manifest to represent the web application. A web app must have a manifest to be a PWA. -->
     <link rel="manifest" href="manifest.json" />
 
-    <!-- A stylesheet to design the pages. -->
-    <link href="styles/global.css" rel="stylesheet" />
-
     <script type="module" src="https://unpkg.com/@fluentui/web-components"></script>
```

Comment:
Can we import the components as needed es modules instead of binging in all components?

Reply:
Yes we could, however I'm not a fan of the developer experience of having to register each component you import. I feel like it easy to forget / miss. I realize it's also a small bit of code in the scheme of things too though and that there are probably even ways PWA Studio could help here. Let me know what you think 😊

Reply:
Oh and yes, registerSW.js is necessary, otherwise there is a console error in dev mode

Reply:
> Yes we could, however I'm not a fan of the developer experience of having to register each component you import. I feel like it easy to forget / miss. I realize it's also a small bit of code in the scheme of things too though and that there are probably even ways PWA Studio could help here. Let me know what you think 😊

Yeah, I know what you mean, I'm not a fan of registering each individual component either. I guess it's a trade of between convenience and performance. I wish there was a way we could do build time analysis and auto import the needed components.